lib/actions/app.coffee

```diff
@@ -100,9 +100,9 @@ exports.info =
 	permission: 'user'
 	action: (params, options, done) ->
 		resin.models.application.get(params.name).then (application) ->
-			console.log visuals.widgets.table.vertical application, [
+			console.log visuals.table.vertical application, [
+				'$' + application.app_name + '$'
```

Comment:
You should use CoffeeScript interpolation feature instead of manually concatenating the strings:

```coffee
"$#{application.app_name}$"
```